test/helpers/assert.js

```diff
@@ -44,8 +44,8 @@ chai.util.addMethod(chai.assert, 'notEmits', function (receipt, eventName, args
   this.isUndefined(event, `Expected that event "${eventName}" with args ${args}  wouldn't be emitted, but it was.`)
 })
 
-chai.util.addMethod(chai.assert, 'reverts', async function (receipt, reason) {
-  await assertRevert(receipt, reason)
+chai.util.addMethod(chai.assert, 'reverts', async function (receipt, reason, customErrorArgs) {
+  await assertRevert(receipt, customErrorArgs !== undefined ? `${reason}(${customErrorArgs.join(', ')})` : reason)
```

Comment:
have `assert.revertsWithCustomError()` already :thinking:

Reply:
Yep, but regular `reverts()` work perfectly with custom errors, so I think we can remove `revertsWithCustomError()` eventually. But the introduced change in the `reverts()` method solves a bit different problem: it allows passing arguments for custom errors as standalone parameters. It might be useful when interpolated error string becomes too long. For example, instead of writing:
```js
await assert.reverts(
    foo.test(),
    `VeryLongErrorMessageWithManyCustomArguments(${firstArgumentWithLongName},${secondArgumentWithLongName},${thirdArgumentWithLongName})`
)
```
might be rewritten in the below form:
```js
await assert.reverts(
    foo.test(),
    'VeryLongErrorMessageWithManyCustomArguments',
    [firstArgumentWithLongName, secondArgumentWithLongName, thirdArgumentWithLongName]
)
```
what improves the reliability of the test (no long strings and horizontal scrolling).